.cirrus.yml

```
@@ -1,95 +1,114 @@
+    memory: 24G
   matrix:
-    # - name: 14-CURRENT
-    #   freebsd_instance:
-    #     image_family: freebsd-14-0-snap
-    - name: 13.2-RELEASE
+    - name: 14.2-RELEASE
       freebsd_instance:
-        image_family: freebsd-13-2
+        image_family: freebsd-14-2
 
 task:
```

YAMLlint (1.35.1)

[error] duplication of key "task" in mapping

(key-duplicates)